packages/types/src/events/assistant.ts

```diff
@@ -0,0 +1,29 @@
+export interface AssistantThreadStartedEvent {
+  type: "assistant_thread_started";
+  assistant_thread: {
+      user_id: string;
+      context: { 
+        channel_id: string; 
+        team_id: string; 
+        enterprise_id: string;
+      };
+      channel_id: string;
+      thread_ts: string;
+  };
+  event_ts: string;
+}
+
+export interface AssistantThreadContextChangedEvent {
+  type: "assistant_thread_context_changed";
+  assistant_thread: {
+    user_id: string;
+    context: { 
+      channel_id: string; 
+      team_id: string; 
+      enterprise_id: string;
```

Comment:
It seems the enterprise_id can be null if the workspace is not under an enterprise grid org.
```suggestion
      enterprise_id: string | null;
```